R/proportional_hazards_data.R

```diff
@@ -213,17 +195,161 @@ make_proportional_hazards_glmnet <- function() {
 #'
 #' Not to be used directly by users
 #'
+#' @param formula The model formula.
+#' @param data The data.
 #' @inheritParams glmnet::glmnet
 #' @param ... additional parameters passed to glmnet::glmnet.
 #' @export
 #' @keywords internal
-glmnet_fit_wrapper <- function(x, y, alpha = 1, lambda = NULL, ...) {
-  fit <- glmnet::glmnet(x, y, family = "cox",
+glmnet_fit_wrapper <- function(formula, data, alpha = 1, lambda = NULL, ...) {
```

Comment:
Should we talk about strata here (or in the Details section here)? 

This is more of a developer function but it would be safe to add it here.

Reply:
sounds good, I've move comments around